contrib/etc/mariadb_environment

```diff
@@ -10,3 +10,36 @@ export MYSQL_PASSWORD=${MYSQL_PASSWORD:-}
 if [[ -z "$MYSQL_ROOT_PASSWORD" ]]; then
   export MYSQL_RANDOM_ROOT_PASSWORD="yes"
 fi
+
+export MYSQL_DEFAULTS_FILE=${MYSQL_DEFAULTS_FILE:-/opt/app-root/etc/my.cnf}
+export MYSQL_BINLOG_FORMAT=${MYSQL_BINLOG_FORMAT:-STATEMENT}
+export MYSQL_LOWER_CASE_TABLE_NAMES=${MYSQL_LOWER_CASE_TABLE_NAMES:-0}
+export MYSQL_MAX_CONNECTIONS=${MYSQL_MAX_CONNECTIONS:-151}
+export MYSQL_FT_MIN_WORD_LEN=${MYSQL_FT_MIN_WORD_LEN:-4}
+export MYSQL_FT_MAX_WORD_LEN=${MYSQL_FT_MAX_WORD_LEN:-20}
+export MYSQL_AIO=${MYSQL_AIO:-1}
+export MYSQL_MAX_ALLOWED_PACKET=${MYSQL_MAX_ALLOWED_PACKET:-200M}
+export MYSQL_TABLE_OPEN_CACHE=${MYSQL_TABLE_OPEN_CACHE:-400}
+export MYSQL_SORT_BUFFER_SIZE=${MYSQL_SORT_BUFFER_SIZE:-256K}
+
+# Auto-tuning
+if [ -z "${MEMORY_LIMIT_IN_BYTES:-}" ]; then
+    key_buffer_size='32M'
+    read_buffer_size='8M'
+    innodb_buffer_pool_size='32M'
+    innodb_log_file_size='8M'
+    innodb_log_buffer_size='8M'
+else
+    key_buffer_size="$(python -c "print(int((${MEMORY_LIMIT_IN_BYTES}/(1024*1024))*0.1))")M"
```

Comment:
See https://github.com/ausnimbus/s2i-nodejs/blob/master/contrib/etc/node_environment#L29, also you shouldn't need python to do this